book/src/drivers.md

```diff
@@ -97,6 +97,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [XSKY-EBS](https://xsky-storage.github.io/xsky-csi-driver/csi-block.html) | `csi.block.xsky.com` | v1.0 | A Container Storage Interface (CSI) Driver for XSKY Distributed Block Storage (X-EBS) | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
 [XSKY-EUS](https://xsky-storage.github.io/xsky-csi-driver/csi-fs.html) | `csi.fs.xsky.com` | v1.0 | A Container Storage Interface (CSI) Driver for XSKY Distributed File Storage (X-EUS) | Persistent | Read/Write Multiple Pods | Yes | 
 [Vault](https://github.com/kubevault/csi-driver) | `secrets.csi.kubevault.com` | v1.0 | A Container Storage Interface (CSI) Driver for mounting HashiCorp Vault secrets as volumes. | Ephemeral | N/A | N/A | 
+[Veritas InfoScale Volumes](https://www.veritas.com/solution/virtualization/containers.html) | `org.veritas.infoscale` | v1.2 | A Container Storage Interface (CSI) Driver for Veritas InfoScale volumes | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning
```

Comment:
Can you provide a link that shows how to use your CSI driver?

Reply:
We have updated the same web page with CSI capabilities and description for Veritas driver. Is this enough?

Reply:
We need something more concrete, i.e., how to deploy the CSI driver, how to create a PVC using the driver, etc.

Reply:
We have added documentation for CSI capabilities and also added a link to detailed documentation for IS CSI under the heading on the same webpage: "Introduction to "InfoScale support for Kubernetes".

Reply:
I'd suggest that the detailed doc for CSI driver be shown as the first link and the high level product intro be shown as the second link.

[Veritas InfoScale Volumes](https://www.veritas.com/support/en_US/doc/145798412-145798468-1) | `org.veritas.infoscale` | v1.2 | A Container Storage Interface (CSI) Driver for [Veritas InfoScale volumes](https://www.veritas.com/solution/virtualization/containers.html) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning

Reply:
Since the URL to "release specific CSI driver documentation" will change with every Veritas IS release, we are using the landing page which has overview of CSI driver and Veritas IS and detailed link to "release specific CSI driver documentation" embedded in this. If we make release specific CSI driver link as landing page, this link will have to be updated for every release to reflect latest Veritas IS release documentation even when no changes happen to CSI driver. 
Updating URL's internally is faster than making changes to k8s CSI docs :)

Reply:
Usually updating CSI docs is pretty straight-forward.  Since this is the first time for this driver, it takes some time to sort things out.  I somehow feel this link "Introduction to InfoScale support for Kubernetes" is hidden in that high level doc.  It will be better if it is easier for users to find it.  Is your CSI driver located on Github by any chance?  It's not required.  Just checking.  For many drivers in this doc, the first link points to the CSI driver github repo which has information on how to use the driver and the second link points to a high level product intro.

Reply:
I'm going to merge this now as information on how to use the CSI driver can be found in the doc.  You can submit a followup PR if you want to update this.